src/status_im2/common/standard_authentication/enter_password/view.cljs

```diff
@@ -0,0 +1,80 @@
+(ns status-im2.common.standard-authentication.enter-password.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [utils.re-frame :as rf]
+            [status-im2.common.standard-authentication.auth-utils :as auth-utils]
+            [utils.security.core :as security]
+            [status-im2.common.standard-authentication.enter-password.style :as style]
+            [quo2.theme :as quo.theme]))
+
+
+
+(defn- view-internal
+  [{:keys [theme on-enter-password button-label]}]
+  (let [entered-password                    (atom "")
```

Comment:
sorry, I'm a bit confused what you mean here. Could you clarify with a code snippet example? 🙏

Reply:
Jamie, Icaro wants to say that this atom will be recreated on each rerender since this is in the render function. To put it outside of it, just use a form-2 component:

```clojure
(defn view [initial-props]
  (let [your-atom (atom "")] ; outside render fn
    (fn [props] ; this is the render function
      [:<>] ; hiccup code
    )
  ))
```

Reply:
thanks!